drv/sidecar-front-io/src/transceivers.rs

```diff
@@ -687,6 +688,9 @@ impl PortI2CStatus {
     pub fn new(status: u8) -> Self {
         // Use QSFP::PORT0 for constants, since they're all identical
         Self {
+            stretching_seen: (status
+                & Reg::QSFP::PORT0_STATUS::STRETCHING_SEEN)
+                != 0,
```

Comment:
wow `rustfmt` is mangling this expression

Very optional, but could we get less terrible formatting if we imported `reg::QSFP::PORT0_STATUS`, then did

```
(status & PORT0_STATUS::STRETCHING_SEEN) != 0
```